doc/release/upcoming_changes/15715.new_feature.rst

```diff
@@ -0,0 +1,4 @@
+``numpy.linalg.multi_dot`` now accepts an out argument
```

Comment:
```suggestion
`numpy.linalg.multi_dot` now accepts an out argument
```
Single backticks should create a link

Reply:
@eric-wieser Added in 841b2259c037dda7095d84c76b5a5010f8012caa